### PR TITLE
Feature/error handling

### PR DESCRIPTION
**Pull request by saraho1123**
## Description

We forgot to add error handling to for the getSingleMovie api call. We added a catch and conditional rendering for a single movie, using the Error component.

## Motivation and Context

To display something other than a blank page to the user if the api response is an error.

## How Has This Been Tested?

Our previous test for Error still works. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Tickets:
#38

### CODE REVIEW

File: src/Error/Error.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import './Error.css';
 
 const Error = ({ error }) => {
   return(
-    <h1>{error}</h1>
+    <h1 className="error-message">{error}</h1>
```
Comment (saraho1123) [inline src/Error/Error.js:6]: We tried adding some text here, but then our test broke. Maybe we can come back to this! :)

File: src/SingleMovie/SingleMovie.js [hunk 2]
```diff
@@ -21,9 +26,18 @@ class SingleMovie extends Component {
...
 
   render() {
+    if(this.state.error) {
+      return (
+        <Route render={() =>
+          <Error error={this.state.error} />
+        }
+        />
+      )
+    }
```
Comment (saraho1123) [inline src/SingleMovie/SingleMovie.js:40]: This all turned out to be quite a bit easier than I expected!

**Review by saraho1123 — COMMENTED** [review]
We did it!

File: src/SingleMovie/SingleMovie.js (hunk 2 above)
Comment (chadrick-d-dev) [reply src/SingleMovie/SingleMovie.js:40]: I agree. Pretty simple!

**Review by chadrick-d-dev — APPROVED** [review]

**Pull request merged**